README.md

```diff
@@ -40,60 +43,30 @@ Here is the list of SODALITE stack components, with corresponding Docker images
-    ```shell script
-    # $DEPLOY_DIR could be set to either docker-local or openstack
-    export DEPLOY_DIR=docker-local
-    openssl genrsa -out $DEPLOY_DIR/modules/docker/artifacts/ca.key 4096
-    openssl req -new -x509 -key $DEPLOY_DIR/modules/docker/artifacts/ca.key -out $DEPLOY_DIR/modules/docker/artifacts/ca.crt
-    cp $DEPLOY_DIR/modules/docker/artifacts/ca.key $DEPLOY_DIR/modules/misc/tls/artifacts/ca.key
-    cp $DEPLOY_DIR/modules/docker/artifacts/ca.crt $DEPLOY_DIR/modules/misc/tls/artifacts/ca.crt
-    ```
-
-5.  ### Set up inputs for deployment
-    Input YAML files provided  in the repository are not supposed to be used as is, but rather serve as a template to be populated with actual values. In order for SODALITE stack deployment to proceed some of inputs must be defined. Namely 
-    `docker-registry-cert-email-address` used for Docker self signed certificate and Gitlab auth token `XOPERA_GIT_AUTH_TOKEN` that grants access to git repository for TOSCA blueprints. To provide inputs manually edit `input.yaml.tmpl` file and save it as `input.yaml`.
-
-    Openstack deployment of SODALITE stack requires a VM to be instantiated therefore these parameters have to be defined: SSH key, image, flavor and network names plus a comma separated list of security groups.
-    ```yaml
-    ssh-key-name: 
-    image-name: 
-    flavor-name:
-    openstack-network-name: 
-    security-groups:     
-    ```
-    *NOTE: `security-groups` input must include `sodalite-uc`security group, e.g. `default,remote_access,sodalite-uc`*
-6.  ### Run blueprint deployment
-    Go to the folder containing `service.yaml` TOSCA blueprint file (`./docker-local` or `./openstack`) and run the following command `opera deploy -i input.yaml service.yaml`
-7.  ### Install SODALITE IDE
+### Input guidelines
+- no input should be empty
+- password should not have special characters
+- [xOpera GIT config](https://github.com/SODALITE-EU/xopera-rest-api#git-backend-server-optional-recommended) 
+(url, token) should be real, otherwise xOpera REST API will not start. 
+### Ubuntu - local
+Clone the repo and run script.
+```shell script
+git clone https://github.com/SODALITE-EU/iac-platform-stack.git
+cd iac-platform-stack
+./deploy_local.sh deploy
+```
+### Ubuntu - openstack
+Clone repo and run script. OPENRC_PATH is path to file with env vars for OpenStack connection.
+```shell script
+git clone https://github.com/SODALITE-EU/iac-platform-stack.git
+cd iac-platform-stack
+./deploy_openstack.sh undeploy [OPENRC_PATH]
+```
+### Alternative installation
+Preferred method for installation is use of [deploy_local.sh](deploy_local.sh) and [deploy_openstack.sh](deploy_openstack.sh). Alternatively, see [manuall install steps](manual_install.md).
+
+## SODALITE stack configuration
```

Comment:
Instead of  "SODALITE stack configuration" this should be "SODALITE IDE installation and configuration" as this is mostly about IDE configuration - no stack configuration here.